src/common/HTTPRequest.ts

```diff
@@ -45,6 +45,26 @@ export interface ResponseForRequest {
   body: string | Buffer;
 }
 
+/**
+ * Resource types for HTTPRequests as perceived by the rendering engine.
+ *
+ * @public
+ */
+export type ResourceType =
```

Comment:
In my local browser_protocol.pdl file, there are some more `ResourceType`s:
- SignedExchange
- Ping
- CSPViolationReport
- Preflight

Is there a specific reason why they are not included here?